R/learn.R

```diff
@@ -119,9 +122,12 @@ mssql_learn_query <- function() { # taken directly from {datamodelr}
       and ind_col.column_id = cols.column_id
     left outer join sys.systypes [types] on
       types.xusertype = cols.system_type_id
+  where tabs.schema_id = schema_id('%s')
   order by
     tabs.create_date,
-    cols.column_id"
+    cols.column_id",
+    schema
```

Comment:
```suggestion
    DBI::dbQuoteIdentifier(con, schema)
```

Otherwise the SQL is bad and prone to injection if `schema` contains `'`.

Reply:
```suggestion
    gsub("'", "''", schema)
```
This way also schemas containing apostrophes are addressed correctly

Reply:
Maybe:

```suggestion
    unclass(DBI::dbQuoteIdentifier(con, schema))
```

? Let's not reimplement `dbQuoteIdentifier()` here.

Reply:
this doesn't work: what we want for a schema called `SELECT` is `'SELECT'`, and with `unclass(DBI::dbQuoteIdentifier(con, schema))` we get `\"SELECT\"`.

for a schema with name `123fn'321` we get `\"123fn'321\"` from `unclass(DBI::dbQuoteIdentifier(con, "123fn'321"))`, but what we want is `'123fn''321'`.

what do you suggest?